Campaign Quota updates-naming and click balancing.

ejm201 opened the pull request:
Edits to Campaign Quota section of API Docs:

- Added name parameter under creating and updating a Campaign Quota (also updated JSON examples).

- Added is_click_balanced parameter under creating and updating a Campaign Quota (also updated JSON examples).

Tagging @chris-tap and @psymike for review.

ejm201 committed:
Campaign Quota updates-naming and click balancing.

itsmikeq commented on source/includes/_campaign_quota.md:
```diff
@@ -10,6 +10,7 @@ Create a quota for a specific campaign.
 {
   "num_respondents": 32,
   "name":"Age quota",
+  "is_click_balanced": 0,
```
This should be `true` instead of `0`.  `1` and `0` work but we're returning a boolean, so it may be confusing to have an int here

itsmikeq commented on source/includes/_campaign_quota.md:
```diff
@@ -101,6 +105,8 @@ Update a quota for a specific campaign.
 {
   "num_respondents": 50,
+  "name":"Updated Quota Name",
+  "is_click_balanced": 1,
```
This should be `true` instead of `0`.  `1` and `0` work but we're returning a boolean, so it may be confusing to have an int here

ejm201 committed:
Updated _campaign_quota.md and _version.md

ejm201 committed:
additional update to _version.md

ejm201 commented:
Thanks for the feedback @psymike. I removed the int's and added true/false to the examples and made some adjustments to the versioning renaming it to changelog after a discussion with @chris-tap.
